cli/js/lib.deno.ns.d.ts

```diff
@@ -696,7 +696,7 @@ declare namespace Deno {
    * Requires allow-write.
    */
   // TODO(ry) Doesn't check permissions.
```

Comment:
Is this comment still valid? If so, can you fix it and check permissions?

Reply:
Oops sorry I missed this.